src/main/kotlin/org/jitsi/nlj/transform/node/incoming/VideoParser.kt

```diff
@@ -55,7 +55,7 @@ class VideoParser(
             return null
         }
         val encodingDesc = findRtpEncodingDesc(videoPacket) ?: run {
-            logger.warn("Unable to find encoding matching packet! packet=$videoPacket, encodings=${tracks.joinToString(separator = "\n")}")
+            logger.warn("Unable to find encoding matching packet! packet=$videoPacket, encodings=${tracks.joinToString(separator = "\n", limit = 1, truncated = "[${tracks.size - 1} more encodings omitted]")}")
```

Comment:
This shows only a single encoding?

Reply:
At that point, is it useful to show them at all?  I think this defeats the utility of the log message a bit.  What about limiting how often it's printed or something?

Reply:
A single track.  So as I understand how that array works this should do the current behavior for non-Octo, and truncate for Octo I think?